plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/SemgrepReader.java

```diff
@@ -54,34 +54,23 @@ public TestSuiteResults parse(ResultFile resultFile) throws Exception {
         return tr;
     }
 
-    private int translate(int cwe) {
-
+    private CweNumber translate(int cwe) {
         switch (cwe) {
-            case 113: // Header injection;
-            case 200: // Information Leak / Disclosure;
-            case 276: // Incorrect Default Permissions;
-            case 352: // CSRF;
-                break; // Don't care - So return CWE 'as is'
-
             case 78:
-                return CweNumber.COMMAND_INJECTION;
+                return CweNumber.OS_COMMAND_INJECTION;
             case 89:
                 return CweNumber.SQL_INJECTION;
             case 90:
                 return CweNumber.LDAP_INJECTION;
             case 326:
             case 327:
             case 696: // Incorrect Behavior Order
-                return CweNumber.WEAK_CRYPTO_ALGO; // weak encryption
+                return CweNumber.WEAK_CRYPTO_ALGO;
             case 614:
             case 1004:
                 return CweNumber.INSECURE_COOKIE;
-            default:
-                System.out.println(
-                        "INFO: Found following CWE in SemGrep results which we haven't seen before: "
-                                + cwe);
         }
-        return cwe;
+        return CweNumber.lookup(cwe);
```

Comment:
Can we restore what we had so we can flag reported CWEs not yet mapped?

Reply:
As discussed: https://github.com/OWASP-Benchmark/BenchmarkUtils/pull/31#issuecomment-1304863379